site/content/docs/browser-checks/timeouts.md

```diff
@@ -6,16 +6,29 @@ menu:
     parent: "Browser checks"
 ---
 
-All browser checks are capped at **120 seconds**. This means everything in your script needs to happen within those 120 seconds.  
+There are different kinds of timeouts you will encounter while working with Browser checks:
```

Comment:
```suggestion
There are different kinds of timeouts you will encounter while working with Browser Checks:
```

Reply:
I'd do so and am pushing for this in marketing because "Checks" is very checkly specific. But feel free to reject for consistency in this page.